landing/index.js

```diff
@@ -1,11 +1,32 @@
+const checkIsVisible = (elem, fixedElem) => {
+    const elemCenter = elem.offsetTop + elem.clientHeight / 2;
+    const containerTop = window.scrollY;
+
+    return containerTop < elemCenter - fixedElem.clientHeight;
+};
+
 window.onload = () => {
-  fetch('./assets/version.txt').then(res => {
-    if (!res.ok) {
-      throw new Error("Failed to load version file");
-    }
-    return res.text();
-  }).then(version => {
-    document.getElementById('app-version').textContent = version;
-    document.getElementById('app-version-label').hidden = false;
-  }).catch((e) => console.error(e.message));
-}
\ No newline at end of file
+    fetch('./assets/version.txt')
+        .then(res => {
+            if (!res.ok) {
+                throw new Error('Failed to load version file');
+            }
+            return res.text();
+        })
+        .then(version => {
+            document.getElementById('app-version').textContent = version;
+            document.getElementById('app-version-label').hidden = false;
+        })
+        .catch(e => console.error(e.message));
+
+    const header = document.getElementById('header');
+    const headerBtn = document.getElementById('btn-header');
+    const tryBtn = document.getElementById('try-now-button');
+
+    const handleScroll = () => {
+        const isVisible = checkIsVisible(tryBtn, header);
```

Comment:
All small screens there are two "Try now" buttons when scrolling down:
![image](https://github.com/veraPDF/verapdf-webapp-gui/assets/62102118/bb0f8afa-ae80-450c-bdcd-2b8445823db3)

Let's hide the `tryBtn` if it's not visible